JuliaGPU/session1/Animesh/q2.jl

```diff
@@ -3,10 +3,10 @@ if n>0
   println(n)
   while(n!=1)
     if n%2==0
-       n = n/2
+       global n = n/2
```

Comment:
This operation changes ```n``` to ```Float``` but the outputs are always integers. So could you print out only integers values? (no decimals needed)